src/AutoUI/Filters/PersistentFilters.tsx

```diff
@@ -158,7 +156,18 @@ export const loadRulesFromUrl = (
 
 				// In case of invalid signatures, remove search params to avoid Errors.
 				if (isSignaturesInvalid) {
-					history.replace({ search: '' });
+					if (
+						history != null &&
+						typeof history === 'object' &&
+						'replace' in history &&
+						typeof history.replace === 'function'
+					) {
+						// react-router-dom v5 history object
+						history?.replace?.({ search: '' });
+					} else if (typeof history === 'function') {
+						// react-router-dom v6 navigate function
+						history({ search: '' }, { replace: true });
+					}
```

Comment:
NIT: since this part is repeated multiple times, we can probably make a util that handle these cases passing `push` or `replace` as an arguments